global.json

```diff
@@ -2,4 +2,4 @@
   "sdk": {
     "version": "2.1.300"
   }
-}
\ No newline at end of file
+}
```

Comment:
Can we please undo the diff here, to eliminate it from git history? A quick way to do that is `git checkout master -- global.json` and committing it - then that commit will zero out in the squash of the PR later :)